_im-plugin/ism/policies.md

```diff
@@ -438,6 +439,120 @@ Parameter | Description | Type | Required
+        "description": "Sample rollup" ,
+        "default_state": "rollup",
+        "states": [
+            {
+                "name": "rollup",
+                "actions": [
+                    {
+                        "rollup": {
+                            "ism_rollup": {
+                                "description": "Creating rollup through ISM",
+                                "target_index": "target",
+                                "page_size": 1000,
+                                "dimensions": [
+                                    {
+                                        "date_histogram": {
+                                            "fixed_interval": "60m",
+                                            "source_field": "order_date",
+                                            "target_field": "order_date",
+                                            "timezone": "America/Los_Angeles"
+                                        }
+                                    },
+                                    {
+                                        "terms": {
+                                            "source_field": "customer_gender",
+                                            "target_field": "customer_gender"
+                                        }
+                                    },
+                                    {
+                                        "terms": {
+                                            "source_field": "day_of_week",
+                                            "target_field": "day_of_week"
+                                        }
+                                    }
+                                ],
+                                "metrics": [
+                                    {
+                                        "source_field": "taxless_total_price",
+                                        "metrics": [
+                                            {
+                                                "sum": {}
+                                            }
+                                        ]
+                                    },
+                                    {
+                                        "source_field": "total_quantity",
+                                        "metrics": [
+                                            {
+                                                "avg": {}
+                                            },
+                                            {
+                                                "max": {}
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        }
+                    }
+                ],
+                "transitions": []
+            }
+        ]
+    }
+}
+````
+
+#### Field parameters
+
+The following field parameters are required:
```

Comment:
The following request fields

I would just explicitly state in each parameter whether it's required or optional.

Reply:
done